lib/salad_ui/input.ex

```diff
@@ -24,7 +24,9 @@ defmodule SaladUI.Input do
   attr :field, Phoenix.HTML.FormField, doc: "a form field struct retrieved from the form, for example: @form[:email]"
 
   attr :class, :string, default: nil
-  attr :rest, :global
+  attr :rest, :global,
+    include: ~w(accept autocomplete capture cols disabled form list max maxlength min minlength
+                multiple pattern placeholder readonly required rows size step)
```

Comment:
This list was taken from `core_components`. This fixes compiler warnings when things like `required` and set on the input.

Might make sense to remove some if they are not intended to be used by input. For example, I don't think `multiple` is needed since input does not support `select` type.